### PR TITLE
Healthcheck tests for metrics susbsystem endpoints

### PR DESCRIPTION
**Pull request by shtripat**
Added tests for node, bucket and resource sub systems

### CODE REVIEW

**Comment by shtripat** [comment]
Does mint health check tests run against ldap, oidc inetegrated MinIO or we need to add them explicitly?

**Comment by poornas** [comment]
> Does mint health check tests run against ldap, oidc inetegrated MinIO or we need to add them explicitly?

those are separate from the mint tests

File: run/core/healthcheck/main.go [hunk 1]
```diff
@@ -33,13 +33,16 @@ import (
...
-	livenessPath     = "/minio/health/live"
-	readinessPath    = "/minio/health/ready"
-	prometheusPath   = "/minio/prometheus/metrics"
-	prometheusPathV2 = "/minio/v2/metrics/cluster"
-	timeout          = time.Duration(30 * time.Second)
+	pass                     = "PASS" // Indicate that a test passed
+	fail                     = "FAIL" // Indicate that a test failed
+	livenessPath             = "/minio/health/live"
+	readinessPath            = "/minio/health/ready"
+	prometheusPath           = "/minio/prometheus/metrics"
```
Comment (harshavardhana) [inline run/core/healthcheck/main.go:40]: wwe don't need to test this, we are moving to v3 now - we can happily remove this endpoint.

Comment (shtripat) [reply run/core/healthcheck/main.go:40]: Will remove the one for `/minio/prometheus/metrics`

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**